packages/workflow/src/Interfaces.ts

```diff
@@ -733,22 +763,46 @@ export interface RequestHelperFunctions {
 	): Promise<any>;
 	requestWithAuthenticationPaginated(
 		this: IAllExecuteFunctions,
-		requestOptions: OptionsWithUri,
+		requestOptions: IRequestOptions,
 		itemIndex: number,
 		paginationOptions: PaginationOptions,
 		credentialsType?: string,
 		additionalCredentialOptions?: IAdditionalCredentialOptions,
 	): Promise<any[]>;
 
+	/**
+	 * @deprecated Use .httpRequest instead
+	 * @see RequestHelperFunctions.httpRequest
+	 */
+	request(uriOrObject: string | IRequestOptions, options?: IRequestOptions): Promise<any>;
```

Comment:
I'm pretty sure `httpRequest` and `httpRequestWithAuthentication` can do everything these old methods can do, but with better typing and cleaner code, so I think marking deprecated is OK, but correct me if I'm wrong.